### PR TITLE
feat: add new dataset for map overlay

### PR DESCRIPTION
**Pull request by pdmnyberg**
This branch will:
- Update instructions on how to add new data overlay datasets
- Add a new data overlay dataset for Yeast GEM

To test:
- Read documentation and make sure it makes sense
- Use dataset in MetAtlas by `build-stack && start-stack`

### CODE REVIEW

File: DATA_OVERLAY.md [hunk 1]
```diff
@@ -57,13 +58,19 @@ As shown by the figure below, the histogram of scaled data is more evenly distri
 
 ![Histogram of HPA scaled data](./assets/img/hpaRNA-TPM-scaleddata-histogram.png)
 
+
+### Linear distribution of label colors
+
+When a dataset contains labels a simplay way to assign values is to split the range `[0,1]` in `N` parts with a fixed step size of `1/N`, where `N` is the number of labels. Then for each entry calculate `scaled_data = indexOf(label)/N`, where `label` is the label of the entry.
```
Comment (perjo) [inline DATA_OVERLAY.md:64]: _Simplay_ - is this a typo or a term I do not know?

Comment (pdmnyberg) [reply DATA_OVERLAY.md:64]: Yes. Thank you!

**Review by perjo — APPROVED** [review]
Looks good to me!

**Comment by KattisLej** [comment]
Starting review

**Pull request merged**